internal/component/database_observability/README.md

```diff
@@ -430,6 +428,24 @@ prometheus.exporter.postgres "integrations_postgres_exporter_<your_DB_name>" {
 database_observability.postgres "postgres_<your_DB_name>" {
   data_source_name  = local.file.postgres_secret_<your_DB_name>.content
   forward_to        = [loki.relabel.database_observability_postgres_<your_DB_name>.receiver]
+
+  // OPTIONAL: enable collecting samples of queries with their execution metrics. The sql text will be redacted to hide sensitive params.
+  enable_collectors = ["query_samples"]
+
+  // OPTIONAL: if `query_samples` collector is enabled, you can use
+  // the following setting to disable sql text redaction (by default
+  // query samples are redacted).
+  query_samples {
+    disable_query_redaction = true
+  }
+
+  // OPTIONAL: provide additional information specific to the Cloud Provider
+  // that hosts the database to enable certain infrastructure observability features.
+  cloud_provider {
+    aws {
+      arn = "your-rds-db-arn"
+    }
+  }
```

Comment:
This is not yet implemented for postgres, but a followup in our todo list so I thought we might just add it now so we don't forget.